### PR TITLE
Adding new developer to pom

### PR DESCRIPTION
**Pull request by amitvc**
Added myself to developer section in pom.xml

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,6 +75,11 @@
             <name>Francesco Caliumi</name>
             <organization>Diennea</organization>
         </developer>
+        <developer>
+            <id>amitvc</id>
+            <name>Amit Chavan</name>
+            <organization>Diennea</organization>
```
Comment (eolivelli) [inline pom.xml]: I think this is not a good value, as you are not an "employee" of Diennea.

This field is optional, you can drop the <organization> tag

File: pom.xml [hunk 2]
```diff
@@ -75,6 +75,11 @@
...
+        </developer>
     </developers>
     <issueManagement>
         <url>https://dev.herddb.org/jira</url>
```
Comment (eolivelli) [inline pom.xml]: can you also please fix this value ?
we moved our issue tracked to github

File: pom.xml (hunk 1 above)
Comment (amitvc) [reply pom.xml]: Ok will do.

File: pom.xml (hunk 2 above)
Comment (amitvc) [reply pom.xml]: ok

**Comment by eolivelli** [comment]
@amitvc feel free to push the "merge button" as soon as CI is green

**Pull request merged**